examples/guestbook/guestbook_ns/frontend_service.json

```diff
@@ -0,0 +1,17 @@
+{
+  "kind": "Service",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "frontend",
+    "labels": {},
```

Comment:
By convention, we should apply the same labels to the service as those in its selector.